blog/2023-09-08-newsletter.md

```diff
@@ -0,0 +1,185 @@
+
+**[Discover more about Pandata here](https://github.com/panstacks/pandata)**.
+
+---
+
+## Releases
+
+**[Conda 23.7.2](https://conda.org/blog/2023-07-28-july-2023-releases#changes-in-conda-237023712372)**: Introduces pre- and post-command plugin hooks, improved subcommand plugin infrastructure, and a health check for altered packages in environments.
+
+[**Conda Build 3.26.0**](https://conda.org/blog/2023-07-28-july-2023-releases#changes-in-conda-237023712372): Logs extra-meta data for package verification, adds pip to env-doc make command, and fixes various bugs related to handling unknown binaries and Git cloning with submodules.
+
+[**constructor 3.4.5**](https://github.com/conda/constructor/releases/tag/3.4.5): Constructor 3.4.5 enhances support for .yaml and .txt files in the environment_file key while ensuring accurate conda metadata in installer creation from existing environments on Windows.
+
+[**conda-libmamba-solver 23.7.0**](https://github.com/conda/conda-libmamba-solver/releases/tag/23.7.0): Libmamba Solver 23.7.0 introduces local channel integrations in conda-build, fixes for namespaced settings, CLI argument handling, authentication in package downloads, and improved handling of channels in conda-lock, among other enhancements and bug fixes.
+
+[**conda-lock 2.1.2**](https://github.com/conda/conda-lock/releases/tag/v2.1.2): conda-lock 2.1.2 resolves Pydantic v2 warnings, implements auth stripping for private PyPi packages, and enhances compatibility with Mamba>=1.4.6. Additionally, it includes infrastructure improvements, dependency updates to pydantic 2.0, and the removal of deprecated pkg_resources, among other changes.
+
+[**conda-package-streaming 0.9.0**](https://github.com/conda/conda-package-streaming/releases/tag/v0.9.0)
+
+[**conda-package-handling 2.2.0**](https://github.com/conda/conda-package-handling/releases/tag/2.2.0)
+
+[**Pixi**](https://github.com/prefix-dev/pixi/releases/tag/v0.2.0): Pixi 0.2.0 introduces a new "pixi search" command for package searching and target-specific tasks. It also includes several bug fixes, installation improvements, and enhancements like a reduction in binary size, updated banner, and contribution guidelines.
+
+---
+
+## [Job still Open at sfbi!](https://www.sfbi.fr/emplois/offre/202308020700-cdd-ingenieur-e-en-developpement-logiciel)
+
+**Job Role**: Software Development Engineer
+
+**Contract Type**: Fixed-Term (12 months)
+
+**Location**: Versailles, France
+
+**Starting Date**: October 1, 2023
+
+**Education**: Bac+5 / Master's Degree
+
+Join in revolutionizing genomics research! They're looking for a Software Development Engineer to enhance our flagship tool, REPET. 
+
+**Your mission**: improve portability and scalability. If you have skills in Python, conda, Snakemake, and more, apply by September 20, 2023. [**Learn more**](https://www.sfbi.fr/emplois/offre/202308020700-cdd-ingenieur-e-en-developpement-logiciel).
+
+---
+
+<Image img={yt} alt="Image of the talk on Youtube"/>
```

Comment:
I think it would be better to instead embed the YouTube video as an iframe. Did you experiment with that approach?

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/d40tBcqopAI?si=NN62GyGLvbEgzqpC" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Reply:
For some reason, the iframe tag isn't working. See if the workaround works for you!

Reply:
Try renaming the file to have the suffix ".mdx". I have already embedded an iframe elsewhere on the website: https://github.com/conda-incubator/conda-dot-org/blob/main/community/calendar.mdx

Reply:
yup, that worked!
Thanks !